lib/movement_problems/coordinate.py

```diff
@@ -0,0 +1,39 @@
+class Coordinate:
+    """
+        A class to calculate coordinates.
```

Comment:
Start the description of the class on the same line
```suggestion
    """A class to calculate coordinates.
```